src/routes/Department.ts

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+import {
+  createDepartment,
+  readAllDepartment,
+  readDepartment,
+  updateDepartment,
+  deleteDepartment,
+} from "../controllers/Department";
+
+const router = express.Router();
+
+router.post("/department", createDepartment);
+router.get("/get/:departmentId", readDepartment);
+router.get("/getAll", readAllDepartment);
+router.patch("/update/:departmentId", updateDepartment);
```

Comment:
we want to patch ("/department/:departmentId")
if you do it like this /update/id will be for every schema -> duplicate route

Reply:
when a request call to /update/id it will not know which one is it